modules.d/95nvmf/parse-nvmf-boot-connections.sh

```diff
@@ -8,20 +8,29 @@
 # Examples:
 # nvmf.hostnqn=nqn.2014-08.org.nvmexpress:uuid:37303738-3034-584d-5137-333230423843
 # nvmf.discover=rdma:192.168.1.3::4420
+# nvme.discover=tcp:192.168.1.3::4420
+# nvme.discover=tcp:192.168.1.3
 # nvmf.discover=fc:auto
 #
 # Note: FC does autodiscovery, so typically there is no need to
 # specify any discover parameters for FC.
 #
 
+if getargbool 0 rd.nonvmf ; then
+    warn "rd.nonvmf=0: skipping nvmf"
+    return 0
+fi
+
+initqueue --onetime modprobe --all -b -q nvme nvme_tcp nvme_core nvme_fabrics
+
+traddr="none"
+trtype="none"
+hosttraddr="none"
+trsvcid=4420
```

Comment:
I'd rather have immutable default values here that `parse_nvmf_discover()` draws from. Currently, The literals (4420, "none"), are being reused.

Reply:
Ok, I'll get in touch with you for clarification about this one.